docs/reference/index-config.md

```diff
@@ -181,15 +199,44 @@ Quickwit supports nested object as long as it does not contains arrays of object
 }
 ```
 
-## Field name validation rules
+### Field name validation rules
 Currently Quickwit only accepts field name that matches the following rules:
 - do not start with character `-`
 - matches regex `[_a-zA-Z][_\.\-a-zA-Z0-9]*$`
 
 
-## Behaviour with fields are not defined in the config
+### Behaviour with fields are not defined in the config
 Fields in your json document that are not defined in the `index config` will be ignored.
 
 
-## Behaviour with null values or missing fields
+### Behaviour with null values or missing fields
 Fields with `Null` or missing field in your json document will be silently ignored when indexing.
+
+
+
+## Indexing settings
+
+This section describes indexing settings.
+
+| Variable      | Description   | Default value |
+| ------------- | ------------- | ------------- |
+| timestamp_field    | timestamp field `timestamp_field` used for [sharding documents in splits](../overview/architecture.md#splits). | None |
+| commit_timeout_secs | Maximum number of seconds before committing a split since its creation. | 60 |
```

Comment:
I think potential costs should be mentioned here (S3 put requests)

Reply:
Will report that on notion as this part is obsolete now.